src/app/(app)/(home)/search-filters/category-dropdown.tsx

```
@@ -0,0 +1,67 @@
+  const onMouseEnter = () => {
+    if (category.subcategories) {
+      setIsOpen(true);
+    }
+  };
+
+  const onMouseLeave = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    // biome-ignore lint/a11y/noStaticElementInteractions: <explanation>
```

Biome (1.9.4)

[error] failed to parse category "lint/a11y/noStaticElementInteractions"



(suppressions/parse)